### ISSUE TITLE
Compatibility with Create React App

### ISSUE THREAD

**Issue by nickpalmer** [birdofpreyru/babel-preset-svgr#1]
Create React App sets up webpack so that svgr output comes exported as a non-default "ReactComponent" and the path to the svg comes out as the default. So our code looks like:

```
import { ReactComponent as Logo } from '../assets/logo.svg';
```

We want to use a wrapper to do server side rendering using babel-register and so I am hoping there is a way to tell this plugin to make the output match Create React App style output so we don't have to change code anywhere. Is there anyway to support this?

**Comment by birdofpreyru** [comment]
Hi @nickpalmer ,
I just released the preset version `1.0.1`, which mimics Create React App behavior if `mimicCreateReactApp` option is set `true` in the Babel config for the preset. Enjoy!

**Comment by nickpalmer** [comment]
Thanks! You are awesome.
